coordinator/src/orderbook/routes.rs

```diff
@@ -268,7 +271,12 @@ async fn websocket(stream: WebSocket, state: Arc<AppState>) {
         while let Some(local_msg) = local_receiver.recv().await {
             match serde_json::to_string(&local_msg) {
                 Ok(msg) => {
-                    if let Err(err) = sender.send(Message::Text(msg.clone())).await {
+                    if let Err(err) = tokio::time::timeout(
+                        WEBSOCKET_SEND_TIMEOUT,
+                        sender.send(Message::Text(msg.clone())),
```

Comment:
❓how long could it happen that we wait without it? 
i.e. what's the "worst" than can happen when you call `sender.send(...)` and the sender is not available. Can this actually result in long waiting times.

I'm happy to add the timeout regardless :)

Reply:
One note: We have to be careful to avoid scenarios where we stop sending on the server side, but the client actually still thinks they are connected.

I remember we had such kind of scenarios in ItchySats - not entirely sure how we can avoid that, but we are starting to add to connection management on the server side again which might lead to problems on the client side.

Reply:
this PS doesn't add any connection management - this one just makes sure we're not hanging forever with a request.

we might need to be careful when we remove entries from the map, as this should always be communicated to both sides appropriately